crates/autopilot/src/run_loop.rs

```diff
@@ -94,18 +97,27 @@ impl RunLoop {
         solutions.sort_unstable_by_key(|solution| solution.1.score);
         let competition_simulation_block = self.current_block.borrow().number;
 
-        let events: Vec<_> = solutions
-            .iter()
-            .flat_map(|(_, s)| s.orders.iter().map(|o| (*o, OrderEventLabel::Considered)))
-            .collect();
-
         // TODO: Keep going with other solutions until some deadline.
         if let Some((index, solution)) = solutions.last() {
             // The winner has score 0 so all solutions are empty.
             if solution.score == 0.into() {
                 return;
             }
 
+            tracing::info!(url = %self.drivers[*index].url, "revealing with solver");
+            let revealed = match self.reveal(id, &self.drivers[*index]).await {
+                Ok(result) => result,
+                Err(err) => {
+                    tracing::error!(?err, "solver {index} failed to reveal");
```

Comment:
Change the log to `warn`?

Reply:
Yeah, I would go with that. We could also have some alert based on metrics if this problem keeps happening but it seems very far fetched that only this endpoint would fail.